backend/src/main/kotlin/com/tngtech/apicenter/backend/connector/database/entity/VersionEntity.kt

```diff
@@ -1,32 +1,31 @@
 package com.tngtech.apicenter.backend.connector.database.entity
 
-import com.tngtech.apicenter.backend.domain.entity.APILanguage
+import com.tngtech.apicenter.backend.domain.entity.ApiLanguage
 import org.hibernate.search.annotations.ContainedIn
 import org.hibernate.search.annotations.Field
 import org.hibernate.search.annotations.IndexedEmbedded
 import org.springframework.stereotype.Indexed
 import java.util.Date
-import java.util.UUID
 import javax.persistence.Column
 import javax.persistence.EmbeddedId
 import javax.persistence.Entity
-import javax.persistence.GeneratedValue
-import javax.persistence.Id
 import javax.persistence.ManyToOne
 import javax.persistence.MapsId
 import javax.persistence.PrePersist
 import javax.persistence.Table
-import javax.persistence.UniqueConstraint
 
 @Entity
 @Indexed
 @Table
 data class VersionEntity(
-    @EmbeddedId @IndexedEmbedded val versionId: VersionId,
-    @Field @Column(columnDefinition = "TEXT") val content: String,
-    @Field @Column(columnDefinition = "TEXT") val language: APILanguage,
-    @ContainedIn @ManyToOne @MapsId("specificationId") var specification: SpecificationEntity?,
-    var created: Date?
+        @EmbeddedId @IndexedEmbedded val versionId: VersionId,
+        @Field @Column(columnDefinition = "TEXT") val content: String,
+        @Field @Column(columnDefinition = "TEXT") val title: String,
+        @Field @Column(columnDefinition = "TEXT") val description: String,
+        @Field @Column(columnDefinition = "TEXT") val language: ApiLanguage,
+        @Field @Column(columnDefinition = "TEXT") val servers: String,
```

Comment:
This is a list of servers, right? Please do not store a list as a single item in the DB.
For now, we can probably only have one server.

Later, we need a set of them, but correlated with environments and stuff.